main.py

```diff
@@ -17,12 +18,15 @@ def read_item(item_id: int, q: Union[str, None] = None):
     return {"item_id": item_id, "q": q}
 
 @app.post("/orders", response_class=PlainTextResponse)
-def get_order(order_details: order.Order):
-    order.save_order(order_details)
+def get_order(order: Order):
+    order.save_order()
     return f"order saved\n"
 
 @app.get("/price")
-def calculate_price(miles: int,
-                    pallets: int = 0,
-                    packages: int = 0) -> float:
-    return price_calculator.calculate(miles, pallets, packages)
\ No newline at end of file
+def calculate_price(miles: int, pallets: int = 0, packages: int = 0) -> float:
+    return calculate(miles, pallets, packages)
+
+@app.post("/validate_order")
+def get_routes(order: Order) -> bool:
```

Comment:
Can you explain why the route is called validate_order but the name of the function is get_routes? I think it is essentially different things. Please be consistent in naming.

Reply:
Whoops, function name was different originally and I forgot to change the path.